get-started/what-is-shiny.qmd

```diff
@@ -0,0 +1,172 @@
+
+```{shinylive-python}
+# | standalone: true
+# | components: [editor, viewer]
+# | layout: vertical
+# | viewerHeight: 250
+
+from shiny import reactive
+from shiny.express import ui, render, input
+
+ui.input_slider("s1", "Slider 1", min=0, max=10, value=5)
+ui.input_slider("s2", "Slider 2", min=0, max=100, value=50)
+
+
+# This output only reacts to the first slider
+@render.text
+def result():
+    return f"{input.s1()} squared is {input.s1() ** 2}."
+
+
+# This output reacts to both sliders
+@render.text
+def both_sliders_output():
+    return f"{input.s1()} + {input.s2()} is {input.s1() + input.s2()}."
+```
+
+## Jumpstart with templates {#templates}
+
+Here's a full Shiny application in action, complete with [reactivity](/docs/reactive-foundations.qmd) and a [user interface](/docs/ui-overview.qmd)!
+
+::::: {.column-screen-inset .mb-4}
+::: {.hello-output}
+<iframe
+  src="https://gallery.shinyapps.io/template-dashboard/"
+  frameborder="0"
+  width="100%"
+  class="mb-0 card hello-output-iframe"
+></iframe>
+
+<style>
+  .hello-output-iframe {
+    height: 700px;
+  }
+@media only screen and (max-width: 611px) {
+  .hello-output-iframe {
+    height: 2125px;
+  }
+}
+</style>
+:::
+:::::
+
+Hit the ground running with one of our [starter templates](/templates/), like the app above, by using `shiny create`.
+
+There are templates for common use cases.
+For example, [data dashboards](templates/dashboard/), [data applications](/templates/database-explorer/), [streaming updates](/templates/monitor-database/), or [data entry](/templates/survey/).
+
+Use the terminal command below to [create and run](create-run.qmd) the same dashboard locally with `shiny create`, starting from a [template](/templates/).
+
+```bash
+shiny create --template dashboard
+```
+
+
+## Extensibility {#extensible}
+
+While Shiny includes everything you need to build an app, it is also built on a foundation of web standards, making it highly [extensible](/docs/custom-component-one-off.qmd).
+In fact, Shiny for Python is built on standard HTML
+Here's an example of how the Shiny action button input returns a HTML `<button>`.
+
+```python
+>>> from shiny import ui
+>>> ui.input_action_button("btn", "Button")
+```
+
+```html
+<button class="btn btn-default action-button" id="btn" type="button">Button</button>
+```
+
+If you're versed in web programming, you can use Shiny to build [one-off custom components](/docs/custom-component-one-off.qmd) that integrate your favorite JavaScript framework directly from Python.
+For more advanced needs, you can develop a [reusable component package](/docs/custom-components-pkg.qmd) or [customize the user interface](/docs/ui-customize) by incrementally [adding JavaScript](/docs/custom-component-one-off), [HTML](/docs/ui-html.html), or even [WebAssembly](/get-started/shinylive.qmd), without needing to learn complex build tools or frameworks.
```

Comment:
There are three different linking styles here -- use `.qmd` links for all of them?

Reply:
```suggestion
For more advanced needs, you can develop a [reusable component package](/docs/custom-components-pkg.qmd) or [customize the user interface](/docs/ui-customize.qmd) by incrementally [adding JavaScript](/docs/custom-component-one-off.qmd), [HTML](/docs/ui-html.qmd), or even [WebAssembly](/get-started/shinylive.qmd), without needing to learn complex build tools or frameworks.
```